be/src/cloud/cloud_internal_service.cpp

```diff
@@ -17,11 +17,39 @@
 
 #include "cloud/cloud_internal_service.h"
 
+#include <brpc/closure_guard.h>
+
+#include "cloud/cloud_storage_engine.h"
+#include "common/signal_handler.h"
+#include "exec/rowid_fetcher.h"
+
 namespace doris {
 
 CloudInternalServiceImpl::CloudInternalServiceImpl(CloudStorageEngine& engine, ExecEnv* exec_env)
         : PInternalService(exec_env), _engine(engine) {}
 
 CloudInternalServiceImpl::~CloudInternalServiceImpl() = default;
 
+void CloudInternalServiceImpl::multiget_data(google::protobuf::RpcController* controller,
+                                             const PMultiGetRequest* request,
+                                             PMultiGetResponse* response,
+                                             google::protobuf::Closure* done) {
+    bool ret = _light_work_pool.try_offer([request, response, done, this]() {
```

Comment:
It's the same as PInternalServiceImpl::multiget_data. Can you call it?

Reply:
no, they are different interface both inherit from PInternalService

Reply:
done